### ISSUE TITLE
pandas: Does not recognize module `pandas.testing`

### ISSUE THREAD

**Issue by DannyNemer** [microsoft/pylance-release#769]
## Environment
-   Language Server: v2020.12.2
-   OS: macOS v10.15.6
-   Python: v3.7.9
-   Pandas: v1.1.5 (latest)

## Expected behavior

Recognize the module [`pandas.testing`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.testing.assert_frame_equal.html).

## Actual behavior

Pylance does not recognize `pandas.testing`:

<img width=600 src=https://user-images.githubusercontent.com/170023/103043392-9e828500-4531-11eb-8ade-232e075cd95e.png>

## Code Snippet

```python
import pandas as pd

pd.testing.assert_frame_equal(pd.DataFrame(), pd.DataFrame())
```

---

Please let me know how I can improve this bug report because many more will follow (all for typeshed issues). Is this the best repo for such bug reports? (I am following the precedent of Pandas type issues in the release notes.)

**Comment by jakebailey** [comment]
I think there's some question as to whether or not this is considered a part of the pandas API in this manner, as opposed to importing `pandas.testing` directly; IIRC this module is exposed via a side effect of Python imports done in the pandas `__init__.py`. If so, the fix will be to add it as an exported variable in our pandas stubs.

**Comment by gramster** [comment]
As far as I can tell the functions in testing are not documented and so I would probably not export them in the stubs. Our stub file is named `_testing.pyi` though so even importing `testing` will not add them; perhaps we should make some adjustment for that? I don't know how widely used these are.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.5.0, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202150-5-may-2021
